src/pages/content-edit/validator.js

```diff
@@ -99,22 +84,24 @@ export function controller(options) {
 }
 
 export function view(ctrl, options) {
-    var style = "";
+    var classes = css.invalidMessage + " ";
 
     if(!ctrl.invalidInputs.length) {
         return m("div", { style : "display:none;" });
     }
 
     // No transition if we're going from 0 -> 1.
-    style = `opacity:${ctrl.currOpacity};`;
-    style += ctrl.currOpacity === 0 ? `transition: opacity ${ctrl.fadeTime}s ${ctrl.fadeDelay}s linear` : "";
+    classes += ctrl.currOpacity === 0 ? css.delayedHide : css.visible;
```

Comment:
Could both `delayedHide` and `visible` compose from `invalidMessage` so that you don't even need to do string concatenation for the classes?